var/spack/repos/builtin/packages/bcftools/package.py

```diff
@@ -34,9 +34,11 @@ class Bcftools(Package):
     homepage = "http://samtools.github.io/bcftools/"
     url      = "https://github.com/samtools/bcftools/releases/download/1.3.1/bcftools-1.3.1.tar.bz2"
 
+    version('1.4', '50ccf0a073bd70e99cdb3c8be830416e')
     version('1.3.1', '575001e9fca37cab0c7a7287ad4b1cdb')
 
     depends_on('zlib')
+    depends_on('bzip2', when="@1.4")
```

Comment:
I would make these `@1.4:`. Once a dependency is added, it isn't likely to go away, so newer versions will likely depend on it as well. You've confirmed that the older versions don't link to bzip2 at all?